mods/suffocation/init.lua

```diff
@@ -0,0 +1,64 @@
+--[[
+	suffocation when head is inside solid node
+	
+	all other features of playerplus was removed
+
+	PlayerPlus by TenPlus1, modified by AKryukov92
+	Original code can be downloaded here https://notabug.org/TenPlus1/playerplus
+]]
+
+-- get node but use fallback for nil or unknown
+local node_ndef = function(pos)
```

Comment:
The following syntax is preferred:

```suggestion
local function node_ndef(pos)
```

Reply:
`local` is missing. I assume this isn't intended to be a global function.